ovos_persona_server/ollama.py

```
@@ -0,0 +1,373 @@
+            "prompt_eval_count": streaming_prompt_eval_count,
+            "prompt_eval_duration": streaming_prompt_eval_duration,
+            "eval_count": streaming_eval_count,
+            "eval_duration": streaming_eval_duration,
+            "done_reason": done_reason
+        }) + "\n"
+
+    return StreamingResponse(streaming_ollama_chat_response(), media_type="application/json")
+
+
+@ollama_router.post("/generate", response_model=OllamaChatResponse, status_code=status.HTTP_200_OK)
+async def generate_ollama(request_body: OllamaGenerateRequest, persona: Persona = Depends(get_default_persona)) -> \
```

Ruff (0.14.0)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)